docs/team/becoming-member.md

```diff
@@ -55,8 +61,52 @@ This process takes the following steps:
    if they are interested. Don't forget to run them by the {ref}`membership_guidelines`
    page to make sure they understand what they're signing up for.
 
-## Membership Maintenance
+### Membership Maintenance
 
-Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply if they still consider themselves active. If not (or no response is given by a team member), it will be assumed that they have gone inactive. This will help keep the active member list up-to-date.
+Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply within 3 weeks if they still consider themselves active. If not (or no response is given by a team member), it will be assumed that they have gone inactive. This will help keep the active member list up-to-date.
```

Comment:
```suggestion
Every six months, one currently active member should open an issue in the team-compass repo asking all currently active team members to reply within three weeks if they still consider themselves active. If a team member replies no, or does not reply, we will conclude that they are inactive. This will help keep the active member list up to date.
```